core/src/avm2/method.rs

```diff
@@ -301,21 +302,87 @@ impl<'gc> fmt::Debug for NativeMethod<'gc> {
     }
 }
 
+#[derive(Clone, Collect, Debug, Copy, PartialEq, Eq)]
+#[collect(no_drop)]
+pub enum MethodPosition {
+    ClassTrait,
+    InstanceTrait,
```

Comment:
Looking through the rest of the project, it appears that `ClassTrait` just indicates if the stacktrace should include a `$` before the method name. I'm assuming script traits (such as free functions) also get marked with `InstanceTrait`?